ush/python/pygfs/task/nexus_emission.py

```diff
@@ -0,0 +1,486 @@
+        super().__init__(config)
+
+        self.task_config = AttrDict(config)
+        pprint(self.task_config)
+
+        self.AERO_INPUTS_DIR = self.task_config.get('AERO_INPUTS_DIR', None)
+        self.COMOUT_CHEM_INPUT = self.task_config.get('COMOUT_CHEM_INPUT', None)
+
+        # get the nforecast hours - gcdas will use FHMAX and gcafs will use FHMAX_GFS
+        if 'das' in self.task_config['RUN']:
+            nforecast_hours = self.task_config["FHMAX"]
+        else:
+            nforecast_hours = self.task_config["FHMAX_GFS"]
+
+        # Create start date based on SDATE
+        self.start_date = self.task_config["CDATE"]
+        self.total_hrs = nforecast_hours + 3
+        self.end_date = self.task_config["CDATE"] + to_timedelta(f'{self.total_hrs}H')
+
+        logger.info(f'SDATE_GFS: {self.start_date}')
+        logger.info(f'nforecast_hours: {nforecast_hours}')
+        logger.info(f'Computed end_date: {self.end_date} (total_hrs={self.total_hrs})')
+
+        # Create the forecast dates based on start_date and end_date
+        frequency = self.task_config.get("NEXUS_DIAG_FREQ", "Hourly")
+        if frequency == "Hourly":
+            self.forecast_dates = list(rrule(freq=HOURLY, dtstart=self.start_date, until=self.end_date))
+        elif frequency == 'Daily':
+            self.forecast_dates = list(rrule(freq=DAILY, dtstart=self.start_date, until=self.end_date))
+        else:
+            raise WorkflowException(f"Unsupported NEXUS_DIAG_FREQ: {frequency}")
+
+        self.forecast_dates_daily = list(rrule(freq=DAILY, dtstart=self.start_date, until=self.end_date))
+
+        logger.info(f"NEXUSEmissions initialized with start date: {self.start_date}, end date: {self.end_date}")
+
+    @logit(logger)
+    def initialize(self) -> None:
+        """Initialize the work directory and process chemical emissions configuration.
+
+        This method performs the following steps:
+        1. Render the NEXUS configuration files using Jinja2 templates
+           found in `parm/chem/nexus/$NEXUS_CONFIG`
+        2. Sets up template variables for emission configuration
+        3. Creates necessary working directories
+        4. Copies required input files to working directory
+        5. Sets up forecast dates and file paths for each date
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        None
+
+        Raises
+        ------
+        WorkflowException
+            If the YAML template file is not found
+            If required directories cannot be created
+            If file copying operations fail
+
+        Notes
+        -----
+        The method expects the following configuration to be available:
+        - HOMEgfs : str
+            Base directory containing workflow configuration
+        - DATA : str
+            Working directory path
+        - COMOUT_CHEM_INPUT : str
+            Output directory for chemical input files
+        - AERO_EMIS_FIRE_DIR : str
```

Comment:
These are in DCOM, right? @aerorahul do we have a naming convention for variables pointing to DCOM like we have for COM?